vendor/data/codelists/IATI/2_03/activity/policy_markers.yml

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
On this, I think we should mirror what comes from IATI directly, otherwise it could cause confusion. See https://iatistandard.org/reference_downloads/203/codelists/downloads/clv3/json/en/PolicySignificance.json

Reply:
I saw you requested changes on this PR. Is this the part you want to see some changes? I'm not very clear with what I should change or not. 
I did it this way because we don't use all the options from IATI for our users. We have removed the last option that IATI considers because it does only affect a policy marker that BEIS don't work with, and also BEIS had added a custom option that I haven't put on this file because it's not IATI related. 
Do you want me to add the last one even if we don't use it and then modify the code so we don't offer that option to the users on the UI?

Reply:
Yeah, I think that would be best. It kinda chimes with what we did with the Aid Type in this PR https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/706 

Apologies if I got the wrong end of the stick!

Reply:
I think I managed to put everything in the right place. Have a look at the new changes when you have time and let me know what you think. Thanks @pezholio!!

Reply:
Looks good! 👍